### PR TITLE
fix(schedule): several bugfix during integration with task framework

### PR DESCRIPTION
**Pull request by guowl3**
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/dlm/model/DataArchiveParameters.java [hunk 1]
```diff
@@ -72,4 +72,6 @@ public class DataArchiveParameters implements ScheduleTaskParameters {
     private ShardingStrategy shardingStrategy;
 
     private RateLimitConfiguration rateLimit;
+
+    private boolean isFullDatabase = false;
```
Comment (guowl3) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/dlm/model/DataArchiveParameters.java:76]: rename to fullDatabase

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleService.java [hunk 2]
```diff
@@ -538,4 +545,49 @@ public boolean hasRunningTask(Long id) {
...
+                changeSchedule(ScheduleChangeParams.with(v, OperationType.TERMINATE));
+            } catch (Exception e) {
+                log.warn("Terminate schedule failed,scheduleId={}", v, e);
+            }
+        });
+    }
+
+    public Map<ScheduleType, Long> countRunningScheduleByDatasourceIds(Set<Long> datasourceIds) {
+        Map<Long, ScheduleEntity> id2Schedule = scheduleRepository.getEnabledScheduleByConnectionIds(
+                datasourceIds).stream().collect(Collectors.toMap(ScheduleEntity::getId, o -> o));
```
Comment (guowl3) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleService.java:566]: query by organization

File: server/odc-common/src/main/java/com/oceanbase/odc/common/util/MapUtils.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+ */
+public class MapUtils {
+
+    private static final String DEFAULT_ENTRY_SEPARATOR = ",";
+    private static final String DEFAULT_KEY_VALUE_SEPARATOR = "=";
+
+    /**
+     * A util that maps a list of entries into a <K,V> map.
+     */
+    public static <K, V> Map<K, V> newMap(Class<K> keyType, Class<V> valueType, Object... entries) {
```
[the remark this answers is not in the document]
Comment (yizhouxw) [reply server/odc-common/src/main/java/com/oceanbase/odc/common/util/MapUtils.java]: validation added

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleService.java (hunk 2 above)
Comment (guowl3) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleService.java:566]: datasource already belongs to the same organization

**Review by yizhouxw — APPROVED** [review]
lgtm

**Pull request merged**